oktoberfest/data/spectra.py

```diff
@@ -110,33 +136,37 @@ def add_matrix_from_hdf5(self, intensity_data: pd.DataFrame, fragment_type: Frag
         :param intensity_data: intensity sparse matrix
         :param fragment_type: choose predicted, raw, or mz
         """
-        # generate column names and build dataframe from sparse matrix
-        columns = self._gen_column_names(fragment_type)
-        intensity_data.columns = columns
-        self.add_columns(intensity_data)
+        # add sparse matrix of intensities to corresponding layer
```

Comment:
do we use this function anywhere? If not, we can delete this alltogether. Not sure where we would need it. I will check this.

Reply:
Yes this is only used in the from_hdf5 function and will become useless once I implement the anndata read_hdf function.